lang/fr.js

```diff
@@ -39,7 +39,7 @@ export default {
 	"components.input-time-range.endTime": "Heure de fin",
 	"components.input-time-range.errorBadInput": "{startLabel} doit précéder {endLabel}",
 	"components.input-time-range.startTime": "Heure de début",
-	"components.list-item-drag-handle.default": "Réorganiser l'action",
+	"components.list-item-drag-handle.default": "Réordonner l’action de l’élément pour {name}",
```

Comment:
Oh, do we need to update this and `fr-fr` to the normal apostrophes?

Reply:
Good catch -- yeah probably eh

Reply:
ah yup good catch